### PR TITLE
[Proposal] Add keyboard visibility observer.

### PR DESCRIPTION
**Pull request by noppefoxwolf**
# Proposal

We sometimes need to know keyboard visibility.
For example, timing of hide buttons when visible keyboard.

### CODE REVIEW

File: Sources/RxKeyboard.swift [hunk 1]
```diff
@@ -29,7 +29,10 @@ public class RxKeyboard: NSObject {
   /// Same with `visibleHeight` but only emits values when keyboard is about to show. This is
   /// useful when adjusting scroll view content offset.
   public let willShowVisibleHeight: Driver<CGFloat>
-
+  
+  /// An observable visibility of keyboard. Emits keyboard visibility
+  /// when changed keyboard show and hide.
+  public let didChangeVisibility: Driver<Bool>
```
Comment (devxoul) [inline Sources/RxKeyboard.swift]: Looks cool. How about using `isHidden` or `isVisible` instead? I think the meaning of 'didChange' is already included in Observable itself.

Comment (noppefoxwolf) [inline Sources/RxKeyboard.swift]: Thank you for your review.
I think so too. but, I think `isVisible` and `isHidden` are looks like store property.
So, I was also thought that use `Variable`.  But `Variable` is writable value from outside class. Keyboard visibility are dependence to responder, so this idea is not cool.

I want your idea about this issue.

My ideas are here.

・use name of `isVisible` or `isHidden` with `Observable`
・use name of `isVisible` or `isHidden` with `Variable`
・add readonly variable like `RxProperty` for visibility property.

Comment (devxoul) [inline Sources/RxKeyboard.swift]: Variable is not fit in this case. I think it doesn't matter even if it looks like a stored property.

Comment (noppefoxwolf) [inline Sources/RxKeyboard.swift]: OK! I pushed f084bc6, please check it.

**Comment by devxoul** [comment]
Thanks for your contribution :tada:

**Pull request merged**